pcsx2/VUops.cpp

```diff
@@ -349,20 +477,19 @@ static __fi float vuADD_TriAceHack(u32 a, u32 b) {
 	// microVU gets the correct operands and result. The interps likely
 	// don't get it due to rounding towards nearest in other calculations.
 
-	if (0) {
-		// microVU uses something like this to get TriAce games working,
-		// but VU interpreters don't seem to need it currently:
-		s32 aExp = (a >> 23) & 0xff;
-		s32 bExp = (b >> 23) & 0xff;
-		if (aExp - bExp >= 25) b &= 0x80000000;
-		if (aExp - bExp <=-25) a &= 0x80000000;
-		float ret = vuDouble(a) + vuDouble(b);
-		DevCon.WriteLn("aExp = %d, bExp = %d", aExp, bExp);
-		DevCon.WriteLn("0x%08x + 0x%08x = 0x%08x", a, b, (u32&)ret);
-		DevCon.WriteLn("%f + %f = %f", vuDouble(a), vuDouble(b), ret);
-		return ret;
-	}
-	return vuDouble(a) + vuDouble(b);
+	// microVU uses something like this to get TriAce games working,
+	// but VU interpreters don't seem to need it currently:
+
+	// Update Sept 2021, now the interpreters don't suck, they do - Refraction
```

Comment:
Well, the comment itself is fine but I think it's best to remove the old ones as in fact, the old broken code isn't here anymore if you see what I mean :).